src/actions/index.js

```diff
@@ -376,6 +378,14 @@ export function removeCommunityModerator (community, moderatorId, prevProps) {
   }
 }
 
+export function removeCommunityMember (community, userId, cacheId, prevProps) {
+  return {
+    type: REMOVE_COMMUNITY_MEMBER,
+    payload: {api: true, path: `/noo/community/${community.id}/member/${userId}`, method: 'DELETE'},
+    meta: {slug: community.slug, userId, cacheId, prevProps}
```

Comment:
the answer probably has to do with middleware -- say, setting `payload.optimistic` triggers middleware that saves a snapshot of the app state, and sets up a promise chain that reverts to it if the final API action returns with `error` set